mckinseyacademy/xblock-poll
# [SE-4860] fix celery upgrade import

## Description

Celery did remove the `celery.decorators` module, or at least `tasks` is not part of that anymore. Therefore, mimicking the previous behaviour, we have to import the current Celery app and use that task.

## Supporting information

OpenCraft currently upgrading Celery on the edX platform:

* https://github.com/edx/configuration/pull/6586
* https://github.com/edx/edx-platform/pull/29046

### Screenshots

## Sandbox

Under provisioning

## Testing instructions

1. Log in to the instance
2. Create a poll
3. Check the poll is created

## Deadline

None

## Other information

N/A
